src/approaches/nsrt_rl_approach.py

```diff
@@ -37,6 +42,11 @@ def get_name(cls) -> str:
     def learn_from_offline_dataset(self, dataset: Dataset) -> None:
         self._initial_trajectories = dataset.trajectories
         super().learn_from_offline_dataset(dataset)
+        # We need to create a separate RL option learner for each option because
+        # each one will maintain its own unique state associated with the
+        # learning process.
+        CFG.option_learner = CFG.rl_option_learner
```

Comment:
why do we need a separate flag for rl option learner vs option learner? can we just use the existing flag?

also, I think we are implicitly assuming that `CFG` is static within the course of a single run (we should probably enforce this). I'm not sure what will go wrong if we change things like this, but something might.

Reply:
oh looking further, I'm guessing this is actually a temporary placeholder